templates/frag/parseMultipartValue.hbs

```diff
@@ -0,0 +1,11 @@
+{{!--
+Determine the correct value to pass from a multipart property to the FromJsonFunction.
+@param <this> CodegenProperty or CodegenSchema
+--}}
+{{#if (isFileSchema .)}}
```

Comment:
I need you to explain this file to me :-D

Reply:
A multipart request is made up of a `body` containing all name-value pairs and `files` containing all name-file pairs. This file helps determine what value to use at each level of validation (and also prevents broken indentation).

If the value passes
- `isFileSchema`, then the value is a `Express.Multer.File[]` array containing the file object we want;
- `isSchemaSupportsMultipart`, then the value is the `files` part of the request made up of name and `Express.Multer.File[]` values;
- else the value must be the `body` part of the request.